chocolatey/choco-theme
# (#381) Remove Add to Calendar buttons in flyout

## Description Of Changes
This removes the "Add to Calendar" buttons in the left side flyout and
instead only shows a "Learn More" button.

## Motivation and Context
This was causing too much work to maintain.

## Testing
1. Run the website.
2. Open the flyout.
3. The "Unpacking Software" and "Product Spotlight" events no longer have "Add to Calendar" buttons.
4. On chocolatey.org, all "Add to Calendar" buttons on the home page and event page still function normally.

## Change Types Made

## Change Checklist


## Related Issue
* https://github.com/chocolatey/chocolatey.org/issues/304
* https://github.com/chocolatey/choco-theme/issues/381
* https://github.com/chocolatey/home/issues/298
* https://gitlab.com/chocolatey/community-infrastructure/community.chocolatey.org/-/issues/1246
